ckanext/canada/wcms.py

```diff
@@ -304,4 +304,4 @@ def wcms_dataset_comment_count(package_id):
         # upstream and see if Sentry can tell us why.
         logging.exception('KeyError occured while pulling comment count.')
 
-    return count
+    return int(count)
```

Comment:
you can be extra paranoid here. we *really* don't want changes in the drupal db to knock us down
```python
try:
    return int(count)
except ValueError:
    return 0
```